### ISSUE TITLE
More compact output format

### ISSUE THREAD

**Issue by mkolassa** [herzrasen/pkghist#10]
It would be nice to see a more compact output format.  Something that can display each version on a single line.  A CSV format would probably satisfy this request, but also maybe something 'slim' like...

[linux]
2018-09-18 16:50:00 | 4.18.8.arch1-1 | Installed
2018-09-20 15:41:00 | 4.18.9.arch1-1 | Upgraded
2018-12-30 20:56:00 | 4.20.arch1-1 | Upgraded

...just to save some vertical space.  Not sure of the value of padding the output to keep things aligned.

Anyways, just an idea/request.

**Comment by herzrasen** [comment]
Thank you for your input. I think it's a nice additional format to have. I'm gonna dig into it in the next few days.

**Comment by herzrasen** [comment]
I implemented something that will output the following format:

```
|zsh    |2019-09-16 22:19:00|Installed|5.7.1-1      |
|linux  |2019-09-20 06:29:00|Upgraded |5.3.arch1-1  |
|linux  |2019-09-23 21:35:00|Upgraded |5.3.1.arch1-1|
|pkghist|2019-09-27 07:47:00|Upgraded |0.1.1-1      |
|pkghist|2019-09-29 02:06:00|Upgraded |0.2.0-1      |
```
I think that printing the package name on each line makes it easier to parse the output later. 

I hope I will be able to release a new version tomorrow.

**Comment by herzrasen** [comment]
implemented in v0.3.0
